homeassistant/components/sensor/hydroquebec.py

```diff
@@ -34,6 +34,7 @@
 
 REQUESTS_TIMEOUT = 15
 MIN_TIME_BETWEEN_UPDATES = timedelta(hours=1)
+SCAN_INTERVAL = timedelta(hours=1)
```

Comment:
Would it be possible to add a force update or allow us to change the update interval?